### PR TITLE
Allow only superusers to use LogicModules endpoint - PLAT-180

### PR DESCRIPTION
**Pull request by ralfzen**
@jefmoura and I aligned that the whole LogicModule-Endpoint should be accessible by superusers only.

### CODE REVIEW

File: workflow/permissions.py [hunk 1]
```diff
@@ -48,6 +48,15 @@ def has_permission(self, request, view):
         return False
 
 
+class IsSuperUser(permissions.BasePermission):
+    """
+    Only superusers are allowed to access.
+    """
+
+    def has_permission(self, request, view):
+        return request.user.is_superuser
```
Comment (jefmoura) [inline workflow/permissions.py]: It's good to check if the user is authenticated as well

**Review by jefmoura — CHANGES_REQUESTED** [review]

File: workflow/permissions.py (hunk 1 above)
Comment (ralfzen) [reply workflow/permissions.py]: done

**Review by jefmoura — DISMISSED** [review]

**Comment by jefmoura** [comment]
Don't forget to add tests for that @ralfzen

**Comment by ralfzen** [comment]
Tests are added. 🤓

**Review by rafa-munoz — APPROVED** [review]

**Review by jefmoura — APPROVED** [review]

**Pull request merged**